nav_tui.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+import rospy
+import curses
+from curses import panel
+from std_msgs.msg import Bool, Float32, String
+import numpy as np
+from ros_alarms import AlarmListener
+from geometry_msgs.msg import PointStamped
+from roboteq_msgs.msg import Status
+
+class nav_tui(object):
+	'''
+	Text User Interface for diagnostics
+	'''
+
+	def __init__(self, screen):
+		'''
+		Initialization of Tui window and fields that will be assigned by callback functions
+		'''
+		self.y, self.x = screen.getmaxyx()
+		self.window = screen.subwin(self.y, self.x, 0, 0)
+		self.window.keypad(1)
+		self.panel = panel.new_panel(self.window)
+		self.panel.hide()
+		panel.update_panels()
+		self.voltage = 0
+		self.wrench  = None
+		self.LAT = None
+		self.LON = None
+		self.ALT = None
+		self.FL_fault = 0
+		self.FR_fault = 0
+		self.BL_fault = 0
+		self.BR_fault = 0
+		self.motor_topics = ['/FL_motor', '/FR_motor', '/BL_motor', '/BR_motor']
+		self.fault_codes = {1: 'OVERHEAT', 2: 'OVERVOLTAGE', 4: 'UNDERVOLTAGE', 8: 'SHORT_CIRCUIT',
+                            16: 'EMERGENCY_STOP', 32: 'SEPEX_EXCITATION_FAULT', 64: 'MOSFET_FAILURE',
+                            128: 'STARTUP_CONFIG_FAULT'}
+
+		#Will try and access low/critical voltage ros parameters, if none are located default values are used
+		try: 
+            		self.lowThreshold = rospy.get_param('battery-voltage/low')
```

Comment:
`self.lowThreshold = rospy.get_param('battery-voltage/low', default=26)` works

Reply:
I have fixed this in my most recent commit, thanks.